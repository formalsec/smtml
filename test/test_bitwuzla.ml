open Smtml_tests
module Test_solver_params = Test_solver_params.Make (Bitwuzla_mappings)
module Test_bv = Test_bv.Make (Bitwuzla_mappings)
