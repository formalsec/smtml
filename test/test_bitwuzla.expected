(model
  (w 0x00000004)
  (x 0x00000001)
  (y 0x00000002)
  (z 0x00000003))